crates/nu-engine/src/eval.rs

```diff
@@ -470,12 +470,12 @@ pub fn eval_variable(
         let mut output_vals = vec![];
 
         let env_vars = stack.get_env_vars();
-        let env_columns: Vec<String> = env_vars.keys().map(|x| x.to_string()).collect();
-        let env_values: Vec<Value> = env_vars.values().cloned().collect();
+        let env_columns = env_vars.keys();
+        let env_values = env_vars.values();
 
         let mut pairs = env_columns
-            .into_iter()
-            .zip(env_values.into_iter())
+            .map(|x| x.to_string())
+            .zip(env_values.cloned())
```

Comment:
Does it need clone?

Reply:
Yeah, I think you get iterator over `&Value` at first so you need a new copy.